packages/ui/src/views/unified-pipeline-studio/components/panel/unified-pipeline-studio-problems-panel.tsx

```diff
@@ -0,0 +1,23 @@
+import { NoProblemsFound, Problems } from '@components/index'
+
+import { useYamlEditorContext } from '@harnessio/yaml-editor'
+
+export function UnifiedPipelineStudioProblemsPanel({ problems }: { problems: any }): JSX.Element {
+  const { updateCursorPosition } = useYamlEditorContext()
+
+  return problems.length === 0 ? (
```

Comment:
do we need this wrapper just for this condition?
can we not just return `NoProblemsFound` from `<Problems />` if the passed array is empty?